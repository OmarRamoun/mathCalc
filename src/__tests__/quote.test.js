import renderer from 'react-test-renderer';
import Quote from '../pages/Quote';

test('Render quote component', () => {
    const tree = renderer.create(<Quote />);
    expect(tree).toMatchSnapshot();
});