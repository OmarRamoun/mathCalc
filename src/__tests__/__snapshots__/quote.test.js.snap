// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render quote component 1`] = `
<div
  className="text-white col-9"
>
  <p
    className="lead fs-2"
  >
    
  </p>
  <button
    className="btn btn-primary"
    onClick={[Function]}
    type="button"
  >
    Get another one
  </button>
</div>
`;
